src/leo-client-app/src/components/Scheduler.css

```diff
@@ -0,0 +1,120 @@
+/* ScheduleStyles.css */
```

Comment:
remove the css comments that were autogenerated

also use styling from the styles file instead of defining them here